modules/nw-ingress-converting-http-header-case.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * ingress/ingress-operator.adoc
+// * upgrade/
+
+[id="nw-ingress-converting-http-header-case_{context}"]
+= Converting HTTP header case
+
+HAProxy 2.2 down-cases HTTP header names by default, for example, changing `Host: xyz.com` to `host: xyz.com`. If legacy applications are sensitive to the capitalization of HTTP header names, use the Ingress Controller `spec.httpHeaders.headerNameCaseAdjustments` API field for a solution to accommodate legacy applications until they can be fixed.
+
+[IMPORTANT]
+====
+Because {product-title} 4.8 includes HAProxy 2.2, make sure to add the necessary configuration by using `spec.httpHeaders.headerNameCaseAdjustments` before upgrading.
+====
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+As a cluster administrator, you can convert the HTTP header case by entering the `oc patch` command or by using the `HeaderNameCaseAdjustments` field by configuring the Ingress Controller YAML file.
+
+Specify a HTTP header to be capitalized by entering the `oc patch` command.
+
+. Enter the `oc patch` command to change the HTTP `host` header to `Host`:
++
+[source,terminal]
+----
+$ oc -n openshift-ingress-operator patch ingresscontrollers/default --type=merge --patch='{"spec":{"httpHeaders":{"headerNameCaseAdjustments":["Host"]}}}'
+----
+
+.. Annotate the route of the application:
++
+[source,terminal]
+----
+$ oc annotate routes/my-application haproxy.router.openshift.io/h1-adjust-case=true
+----
++
+The Ingress Controller then adjusts the `host` request header as specified.
+
+//Extra example if needed
+////
+* This example changes the HTTP `cache-control` header to `Cache-Control`:
++
+[source,terminal]
+----
+$ oc -n openshift-ingress-operator patch ingresscontrollers/default --type=json --patch='[{"op":"add","path":"/spec/httpHeaders/headerNameCaseAdjustments/-","value":"Cache-Control"}]'
+----
++
+The Ingress Controller adjusts the `cache-control` response header as specified.
+////
+
+Specify adjustments using the `HeaderNameCaseAdjustments` field by configuring the Ingress Controller YAML file.
```

Comment:
This threw me off that the examples didn't do the same thing. If these are two ways to accomplish the same thing, shouldn't the examples have the same end result? 

Or maybe  I'm missing something on why you'd want to do one method over the other, if they do accomplish different things.